mods/persistence/modules/mob/living/carbon/human/human.dm

```diff
@@ -23,25 +27,28 @@
 				QDEL_NULL(home_spawn)
 		forceMove(get_spawn_turf()) // Sorry man. Your bed/cryopod was not set.
 
+/mob/living/carbon/human/setup(species_name, datum/dna/new_dna)
+	//If we're loading from save, go through setup using the existing dna loaded from save
+	if(persistent_id && dna)
+		. = ..(null, dna)
+	else
+		. = ..()
+
 /mob/living/carbon/human/Initialize()
-	if(!persistent_id)
-		return ..()
-	set_species(LOAD_CUSTOM_SV("saved_species"), FALSE)
-	set_bodytype(species.get_bodytype_by_name(LOAD_CUSTOM_SV("saved_bodytype")), FALSE)
 	. = ..()
 	LATE_INIT_IF_SAVED
 
-// /decl/species/create_organs(var/mob/living/carbon/human/H)
-// 	//We don't want to delete the organs we loaded from the save
-// 	if(!H.persistent_id)
-// 		. = ..()
-	
-// 	H.mob_size = mob_size
-
 /mob/living/carbon/human/LateInitialize()
 	. = ..()
 	if(persistent_id)
-		set_move_intent(GET_DECL(LOAD_CUSTOM_SV("saved_move_intent")))
+		set_move_intent(GET_DECL(LOAD_CUSTOM_SV("move_intent")))
+
+		//Apply saved appearance (appearance may differ from DNA)
```

Comment:
Does appearance differing from DNA have any potential in game consequences?

Reply:
I know at one point changing hair colors and haircut would change it on the mob only. 

The larger problem is that there can be changes made to a mob's appearance that probably shouldn't end up in their DNA. (EX: if you dye your hair blue or something) But the current code makes no real distinctions, and isn't coherent either about it. Its like, if it wasn't intended to have permanent and non-permanent changes to those, why would they be stored twice?
Also, using the value from the DNA instead will introduce weirdness from how numbers are stored in the DNA currently.